ob_v2p1/ob-spec-v2p1.html

```diff
@@ -1327,6 +1385,30 @@ <h4>200 GET Profile Response</h4>
       </section>
       <section id="api-responses-post-profile-200">
         <h4>200 POST Profile Response</h4>
+        <table>
+          <thead>
+            <tr>
+              <th>Property</th>
+              <th>Required</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><code>status</code></td>
+              <td>Yes</td>
+              <td>The <a href="#api-envelope-status">Status</a> object describing the success or failure of the request.</td>
+            </tr>
+            <tr>
+              <td><code>results</code></td>
```

Comment:
I think `profile` was the property name that is defined in your initial comment on this PR, not `results`.

Reply:
Yes.